### ISSUE TITLE
Deeply nested blockquotes make the app unresponsive.

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#3034]
Long email threads cause several lag issues on mobile devices because each reply introduces another level of nesting..We should consider collapsing or not nesting the quotes.

Another solution could be To not display quoted email in reply.

**Comment by johnbotris** [comment]
Is it not just large amounts of text, rather than the quoting itself?

**Comment by jowlo** [comment]
We could think about putting blockquotes in expanders. Let's say by default we show the first two/three levels and everything below that we just put in an expander that you have to click open.

I suspect that this stems from very long mail threads where people just reply and i guess noone ever reads the nested stuff anyways. if it is basically not readable (because of borked formatting) we might as well not display it.

**Comment by charlag** [comment]
@jowlo yes I agree we should not display it at all by default but we should do conversation view first

**Comment by ganthern** [comment]
with conversation view out, this might be resolved

**Comment by charlag** [comment]
Still a problem for the editor
